src/pf-notification-drawer/pf-notification-drawer.scss

```diff
@@ -0,0 +1,46 @@
+.drawer-pf {
```

Comment:
Ok, noted in a new issue [#71](https://github.com/patternfly-webcomponents/patternfly-webcomponents/issues/71). I'm fine w/ moving to LESS.